packages/-ember-data/tests/integration/inverse-test.js

```diff
@@ -1,68 +1,77 @@
-import { run } from '@ember/runloop';
-import setupStore from 'dummy/tests/helpers/store';
-
+import { setupTest } from 'ember-qunit';
 import testInDebug from 'dummy/tests/helpers/test-in-debug';
 import { module, test } from 'qunit';
 
-import DS from 'ember-data';
-
-let env, store, User, Job, ReflexiveModel;
-
-const { attr, belongsTo } = DS;
+import Model from '@ember-data/model';
+import { attr, belongsTo } from '@ember-data/model';
 
 function stringify(string) {
   return function() {
     return string;
   };
 }
 
+// @pete_the_pete refer to https://github.com/emberjs/data/commit/e34cb5a37391ce2e5d25401ca189a9bd19a29340
```

Comment:
forgotten?

Reply:
💯

Reply:
shows as resolved but still shows up in the latest diff

Reply:
Still needs removed

Reply:
i think i got it this time.